sda/service.go

```diff
@@ -113,6 +114,13 @@ func RegisterNewService(name string, fn NewServiceFunc) {
 	ServiceFactory.Register(name, fn)
 }
 
+// DeleteNewService will remove the NewServiceFunc from the global store of
+// NewServiceFunc so it can't be initialized again. If the service needs to be
+// initialized again, a new call to RegisterNewService must be made.
```

Comment:
This comment seems a little outdated or not? 
Change to: `DeleteNewService` completely removes the service associated with the passed name.